### ISSUE TITLE
Using shortcut `Cmd/Ctrl-W` on a window with multiple tabs closes the entire window instead of focused tab

### ISSUE THREAD

**Issue by balsa-sarenac** [guillep/pharo-core#16436]
**Bug description**
Using shortcut `Cmd/Ctrl-W` on a window with multiple tabs closes the entire window instead of focused tab

**To Reproduce**
1. Open a senders window for a method and then fill it with tabs (look for implementors and senders of some method)
2. Press `Cmd/Ctrl-W`
3. Entire window is closed

**Expected behavior**
Only the focused tab should be closed.

**Comment by Ducasse** [comment]
Thanks a lot because recently I could not really work with P12 since I was always closing the messageList when I just press ctrl -w to close a pane. Now I got it.

**Comment by guillep** [comment]
We investigated a bit here. The problem seems to be that there is a global shortcut that "closes the top window"

<img width="846" alt="imagen" src="https://github.com/pharo-project/pharo/assets/708322/51fb10da-a5fe-4956-9b38-7a19cea6aa06">

**Comment by guillep** [comment]
We chased the bug up to.

```
FTTableMorph >> initializeShortcuts: aKMDispatcher

	super initializeShortcuts: aKMDispatcher.
	aKMDispatcher attachCategory: #WindowShortcuts
```

This means that all tables, regardless of where they are embedded, will for the closing of the top window.
It's however not clear what would we impact if we remove this.

**Comment by guillep** [comment]
This issue was introduced in https://github.com/pharo-project/pharo/pull/16386 as an attempt to fix https://github.com/pharo-project/pharo/issues/16313.

However, to me this fix does not work, I don't think tables should support window shortcuts.
But we should reopen the original issue if we remove this.

**Comment by hernanmd** [comment]
> This issue was introduced in #16386 as an attempt to fix #16313.
> 
> However, to me this fix does not work, I don't think tables should support window shortcuts. But we should reopen the original issue if we remove this.

Thanks for taking a look in this.

I think the issue report is incomplete. It doesn't mention where should be the *focus*, so now to close the whole window one should click in the SystemWindow. So the problem isn't the Ctrl/CMD+W but where to set the focus after a window is opened or closed.
